src/application/commands/CommandPalette.tsx

```diff
@@ -96,6 +100,30 @@ export function MainCommandMenu({ index }: { index: number }) {
           text: 'New File',
           perform: () => emitEvent('refstudio://menu/file/new'),
         },
+        {
+          icon: <VscLayoutSidebarLeft />,
+          text: 'Move Left',
+          perform: () => {
+            if (activeEditorId) {
```

Comment:
Is there a way to disable or remove the item if there no active editor?

Reply:
I don't know a way of doing that (that's easy). That's definitely something the library can improve in the future.